services/frontend/src/components/NavSidebar/NavSidebar.scss

```diff
@@ -1,6 +1,10 @@
 @import "styles/_colors.scss";
 @import "styles/_dimens.scss";
 
+.project-page .scroll-nav .list-item:last-child {
+    margin-top: $default-spacing*1.5;
```

Comment:
In accordance with the recommended `ScrollContainer` changes, this style should be moved to `ProfileDetails.scss`.